config/opensearch_dashboards.yml

```diff
@@ -321,6 +321,12 @@
 #   AWSSigV4:
 #     enabled: true
 
+# Optional setting that controls the permissions of data source to create, update and delete.
+# "read_only": The data source is readonly for all users including OSD admin.
+# "admin_only": The data source can only be edited by OSD admin.
+# "none": The data source can be edited by all users. Default to "none".
+# data_source.editMode: "none"
```

Comment:
@ruanyl @SuZhou-Joe What do you think about renaming `editMode` to `operateMode`? The `editMode` is a little bit confused to me. Seems editMode only control the data source edit behavior. We still can create new data source.

Reply:
What about:
```
data_source.manageable_by: "all"
data_source.manageable_by: "dashboard_admin"
data_source.manageable_by: "none"
```

and default to `all` to keep backward compatible?

Reply:
@wanglam That's good point, agree with that.
I think what @SuZhou-Joe suggested sounds good to me.